From: Stanislav Malyshev <stas@php.net>
Date: Sun, 15 Mar 2020 19:35:26 -0700
Subject: Update NEWS

(cherry picked from commit c8d21d7728109b0f911033c098cfaeb7438ba1d5)
(cherry picked from commit 03471e31c9b467d1d8d944e44fa009ef247e81bd)
(cherry picked from commit 4844343ac37e8e3ca4d995b1d91fc0f9daf03d5f)
---
 NEWS | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/NEWS b/NEWS
index 78678bc..98a3b4d 100644
--- a/NEWS
+++ b/NEWS
@@ -1,6 +1,16 @@
 PHP                                                                        NEWS
 |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
 
+Backported from 7.2.29
+
+- Core: 
+  . Fixed bug #79329 (get_headers() silently truncates after a null byte) 
+    (CVE-2020-7066) (cmb)
+
+- EXIF:
+  . Fixed bug #79282 (Use-of-uninitialized-value in exif) (CVE-2020-7064) 
+    (Nikita)
+
 Backported from 7.2.28
 
 - Phar:
